.github/workflows/devshell.yml

```diff
@@ -143,7 +143,7 @@ jobs:
         run: ./autogen.sh
 
       - name: configure
-        run: ./configure ${DISTCHECK_CONFIGURE_FLAGS}
+        run: ./configure --disable-all-modules --enable-manpages
```

Comment:
Shouldn't `DISTCHECK_CONFIGURE_FLAGS` use the new `--enable-manpages-install` instead of `--enable-manpages`?

Reply:
No, distcheck needs the manual pages and git does not have them. So in order to successfully generate "make dist", we will need --enable-manpages.

Reply:
Yes, you are right. `DISTCHECK_CONFIGURE_FLAGS` is used when the tarball is created, not just when make tries to build the package from the tarball.
https://www.gnu.org/software/automake/manual/html_node/Checking-the-Distribution.html